modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerRecoveryTest.java

```diff
@@ -239,6 +241,7 @@ public void testTableIgnoredOnRecovery() throws Exception {
     }
 
     @Test
+    @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "false")
```

Comment:
Why it's !colocation only?

Reply:
There is the following line in the test:

```
verify(mvTableStorage, timeout(WAIT_TIMEOUT).times(PARTITIONS)).createMvPartition(anyInt());
```
It makes sure that partitions were started (actually, that their storages were created), but we don't do that for colocation case anymore.

Reply:
Semantically TableManager still recovers tables on startAsync in case of colocation track, thus we check it. Precisly startAsync() calls startTables() that on it's own calls prepareTableResourcesAndLoadToZoneReplica that calls getOrCreatePartitionStorages etc.